lib/ndr_import/table.rb

```diff
@@ -92,6 +92,16 @@ def header_valid?
       @header_valid == true
     end
 
+    # For readability, we should serialise the columns last
+    def encode_with(coder)
+      options = self.class.all_valid_options - ['columns']
+      options.each do |option|
+        value = instance_variable_get("@#{option}")
```

Comment:
we have `attribute_reader` for all options; can we just use `send(option)` instead?